.github/workflows/kind.yml

```diff
@@ -10,9 +10,27 @@ on:
     - master
     - release-*
     - ipv6
+
 jobs:
+  check-changes:
+    name: Check whether tests need to be run based on diff
+    runs-on: [ubuntu-18.04]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    # TODO: change to vmware-tanzu/antrea/ci/gh-actions/has-changes@master when #1188 is merged
+    - uses: antoninbas/antrea/ci/gh-actions/has-changes@add-github-action-to-detect-when-only-documentation-has-changed
+      id: check_diff
+      with:
+        args: docs/* ci/* *.md
```

Comment:
should "ci/*" be included? the scripts and yamls that run kind and upgrade tests are in it, could them be broken without testing?

Reply:
Thanks for catching this. I meant to use `ci/jenkins/*` here. Fixed now.